tlslite/messages.py

```diff
@@ -1680,6 +1680,50 @@ def write(self):
         return self.postWrite(w)
 
 
+class EncryptedExtensions(HandshakeMsg):
+    """Handling of the TLS1.3 Encrypted Extensions message."""
+
+    def __init__(self):
+        super(EncryptedExtensions, self).__init__(
+                HandshakeType.encrypted_extensions)
+        self.extensions = None
+
+    def create(self, extensions):
+        """Set the extensions in the message."""
+        self.extensions = extensions
+
+    def parse(self, parser):
+        """Parse the extensions from on the wire data."""
+        parser.startLengthCheck(3)
+
+        if not parser.getRemainingLength():
+            raise SyntaxError("No list of extensions")
```

Comment:
`SyntaxError` in python is raised when the _python code_ has a syntax error itself. It was incorrectly used in tlslite to indicate errors in parsing of TLS messages, I'm using it in tlslite-ng for backwards compatibility. Also, the message will be carried up to the user application and likely used in error messages, so a descriptive one is useful either way.